applications/nrf_desktop/README.rst

```diff
@@ -955,6 +955,42 @@ To use the nRF Desktop application with your custom board:
    Some options have dependencies that might not be needed when these options are disabled.
    For example, when the LEDs module is disabled, the PWM driver is not needed.
 
+Adding a new motion sensor
```

Comment:
> The first paragraphs don't tell me why I would want to add a new sensor.

You had it in the first paragraph and even made a comment :)
> It is likely that another sensor will be more suitable for a new board.

Reply:
> It is likely that another sensor will be more suitable for a new board.

This doesn't serve this purpose, because as written earlier, it is a supposition, not a statement of fact. This `likely` poses questions like "why will it be more suitable? what is wrong with the NCS sensors? what another sensor should be added?". None of these questions is answered here.